docs/README.md

```diff
@@ -194,13 +194,13 @@ You should see output indicating the successful creation of a database and start
 Now that we have our development environment set up, the next step is to create an index.
 
 ```bash
-forc index new hello-index --namespace my_project && cd hello-index
+forc index new hello-indexer --namespace my_project && cd hello-index
```

Comment:
`forc index new` will create a new directory with whatever string you give, so we should be `cd`-ing into `hello-indexer`.